en/guide/toolchain.md

```diff
@@ -118,30 +118,34 @@ target_link_libraries(your_executable_name
 
 This section assumes that you have already [built and installed the DroneCore C++ Library](../contributing/build.md) and that your example has a **CMakeLists.txt** in its root directory. 
 
-To create and run the app:
+To create and run the app on Linux/macOS:
 1. First create a build directory and run *cmake*: 
    ```bash 
    cd /path/to/your/application/   # Open a prompt and navigate to your application 
    mkdir build && cd build   # Create a build directory and navigate into it
    cmake ..   # Create make files for your current platform
    ```
    > **Tip** Make files for release binaries are created by default. Use the flag `-DCMAKE_BUILD_TYPE=Debug` to build debug binaries:
-  ```
-  cmake -DCMAKE_BUILD_TYPE=Debug ..
-  ```
+   ```
+   cmake -DCMAKE_BUILD_TYPE=Debug ..
+   ```
 1. Build the project
-   * Linux/macOS:
-     ```bash 
-     make -j8
-     ``` 
-   * Windows:
-     ```bash 
-     cmake --build .
-     ``` 
+   ```bash 
+   make -j8
```

Comment:
This is funny because the current examples only contain one cpp file, so you can compile them with only one core anyway 😄 . 8 cores is very optimistic (but not wrong!)

Reply:
Fixed. Just one core, but with comment that you can use more.